src/app.module.ts

```diff
@@ -1,10 +1,40 @@
 import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigType } from '@nestjs/config';
+import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { ProductModule } from './products/product.module';
+import { envVariables, envVariablesValidations } from './shared/config';
```

Comment:
Consider implementing path aliases for your imports. It will solve you a lot of headaches based on absolute/relative imports and also will be easier to read.

Reply:
Done!